sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/integrationtest/BeamSqlComparisonOperatorsIntegrationTest.java

```diff
@@ -281,7 +281,7 @@ public void testIsNullAndIsNotNull() throws Exception {
   }
 
   @Override protected PCollection<Row> getTestPCollection() {
-    RowType type = RowSqlType.builder()
+    Schema type = RowSqlType.builder()
```

Comment:
<!--thread_id:cc_178336155_t; commit:79c95678e593da730ba0472b77304ec1f916245e; resolved:0-->
<!--section:context-quote-->
> **akedin** wrote:
> Now that we have all type info in core, we should have the builder there as well: `Schema.builder().withInt32().build()`

<!--section:body-->
So the "extra" use for this class now is that it has builder methods specifically for SQL types (e.g. timestamp, date, varchar, char), and passes them through CalciteUtils to do the correct translation to schema. Seems less error prone to centralize this rather then to have everyone call into CalciteUtils.